prod-refractr.yml

```diff
@@ -16,6 +16,9 @@ refracts:
   - activations.mozilla.com
   - activations.mozilla.org
 
+  # SE-1683
+- addons.mozilla.org/: addons.mozilla.com
```

Comment:
Hmm, there is already an addons.mozilla.org entry below this. Would it work just as well to add addons.mozilla.com to end of the list at line 30?

Reply:
you're right, this is my bad. and i looked at that block too beforehand

Reply:
done